capa/render/proto/capa.proto

```diff
@@ -204,93 +214,69 @@ message Metadata {
 message MnemonicFeature {
   string type = 1;
   string mnemonic = 2;
-  string description = 3;
+  optional string description = 3;
 }
 
 message NamespaceFeature {
   string type = 1;
   string namespace = 2;
-  string description = 3;
+  optional string description = 3;
 }
 
 message NumberFeature {
   string type = 1;
-  oneof number {
-    Integer v0 = 2;
-    Number v1 = 3;
-  };
-  string description = 5;
+  Number number = 2;  // TODO can/should this be negative?
```

Comment:
i think there are certain cases when we can infer the number might be signed (and/or maybe in .NET its very obvious?) so we should support negative numbers.